cmd/conformance/main.go

```diff
@@ -23,25 +23,33 @@ import (
 	"os"
 	"path"
 	"runtime"
+	"time"
 
 	protobundle "github.com/sigstore/protobuf-specs/gen/pb-go/bundle/v1"
 	protocommon "github.com/sigstore/protobuf-specs/gen/pb-go/common/v1"
+	"google.golang.org/protobuf/encoding/protojson"
 
 	"github.com/sigstore/sigstore-go/pkg/bundle"
 	"github.com/sigstore/sigstore-go/pkg/root"
+	"github.com/sigstore/sigstore-go/pkg/sign"
 	"github.com/sigstore/sigstore-go/pkg/tuf"
 	"github.com/sigstore/sigstore-go/pkg/verify"
 )
 
+var Version string
 var bundlePath *string
 var certPath *string
 var certOIDC *string
 var certSAN *string
+var identityToken *string
 var signaturePath *string
+var staging = false
 var trustedRootPath *string
 
 func usage() {
 	fmt.Println("Usage:")
+	fmt.Printf("\t%s sign --identity-token TOKEN --signature FILE --certificate FILE [--staging] FILE", os.Args[0])
```

Comment:
Is this needed because we have conformance tests for detached metadata?

Reply:
It's part of the [conformance CLI protocol](https://github.com/sigstore/sigstore-conformance/blob/main/docs/cli_protocol.md) and it looks like there are tests that [test signing with detached materials](https://github.com/sigstore/sigstore-conformance/blob/main/test/test_signature_verify.py#L67-L80).